src/main/java/emu/grasscutter/server/game/GameServer.java

```diff
@@ -212,6 +202,23 @@ public void deregisterWorld(World world) {
 		
 	}
 
+	@Override
+	public synchronized void start() {
+		// Schedule game loop.
+		ScheduledExecutorService gameLoop = Executors.newScheduledThreadPool(2);
```

Comment:
Why did you move the game loop and why is it in a thread pool..?

Reply:
Because onTick function in game loop will call `Grasscutter.getPluginManager().invokeEvent(this);`,
but after moving the order of creating plugin manager, it will cause a null point exception because  it is not instantiated in this time。As for thread pool, it's the recommendation of IDEA....

Reply:
Makes sense but theres no need to put the gameloop in a thread pool.

Reply:
okay, I've rolled it back